src/pokete/classes/fight/fight.py

```
@@ -87,12 +93,26 @@ def __call__(self, ctx: Context, providers: list[Provider]):
+                if player == self.providers[0]: # Player
+                    # Filter out IRL distractions (> 30s)
+                    if decision_time < 30.0:
+                        hesitations.append(decision_time)
+
                 match attack_result.result:
                     case Result.ATTACK:
                         attack: Attack = attack_result.attack_value
                         if player == self.providers[0]: # Player
                             unique_moves.add(attack.index)
+                            # Playstyle Scoring
+                            if attack.factor >= 2.0: aggression_count += 1
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)